src/udev/net/link-config.c

```diff
@@ -1314,6 +1374,39 @@ int config_parse_wol_password(
         return 0;
 }
 
+int config_parse_rps_cpu_mask(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        LinkConfig *config = ASSERT_PTR(userdata);
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (streq(rvalue, "all")) {
+                r = cpu_mask_add_all(&config->rps_cpu_mask);
+                if (r < 0)
+                        log_syntax(unit, LOG_WARNING, filename, line, r,
+                                   "Failed to create CPU affinity mask representing \"all\" cpus, ignoring: %m");
+        } else {
+                r = parse_cpu_set_extend(rvalue, &config->rps_cpu_mask, true, unit, filename, line, lvalue);
```

Comment:
our newest coding style suggests that if you pass an opaque value to some function you prefix it with the parameter name as comment to make it easier to read. or in other words, change the third arg to `/* warn= */ true`